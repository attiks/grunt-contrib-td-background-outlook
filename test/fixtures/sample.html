<html>
  <head>
  </head>
  
  <body>
    <h1>Hello</h1>
    <table>
      <tr>
        <td width="455" height="287" bgcolor="#ff0000" background="//cdn.example.com/stuff/bar.jpg">
          Hello
        </td>
        <td width="155" background="//example.com/good/bar.jpg">
        </td>    
        <td>
          A normal td
        </td>
      </tr>
    </table>
  </body>
</html>
