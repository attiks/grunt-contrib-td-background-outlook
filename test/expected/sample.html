<html>
  <head>
  </head>
  
  <body>
    <h1>Hello</h1>
    <table>
      <tr>
        <!--[if gte mso 9]><v:rect xmlns:v="urn:schemas-microsoft-com:vml" fill="true" stroke="false" style="width:455px;height:287px;"><v:fill type="tile" src="//cdn.example.com/stuff/bar.jpg" color="#ff0000" /><v:textbox inset="0,0,0,0"><![endif]--><div>
          Hello
        </div><!--[if gte mso 9]></v:textbox></v:rect><![endif]--></td>
        <!--[if gte mso 9]><v:rect xmlns:v="urn:schemas-microsoft-com:vml" fill="true" stroke="false" style="width:155px;"><v:fill type="tile" src="//example.com/good/bar.jpg" color="#ffffff" /><v:textbox inset="0,0,0,0"><![endif]--><div></div><!--[if gte mso 9]></v:textbox></v:rect><![endif]--></td>
        <td>
          A normal td
        </td>
      </tr>
    </table>
  </body>
</html>
